src/app/views/marketplace/marketplace-app.html

```diff
@@ -91,26 +91,134 @@ <h2 class="media-heading">{{vm.app.name}}</h2>
           <!-- End pics/details row -->
         </div>
 
+        <!-- Description row -->
+        <div class="row">
+          <div class="col-md-12">
+            <div class="well top-buffer-1" ng-bind-html="vm.app.description"></div>
+          </div>
+        </div>
+
         <!-- Testimonials row -->
         <div class="row">
-          <div class="col-md-6" ng-repeat="testimonial in vm.app.testimonials | limitTo:2" ng-show="vm.isTestimonialShown(testimonial)">
+          <div class="col-md-12 tabs">
+            <button class="btn" ng-class="{active: vm.rewieTab}" type="button" name="button" ng-click="vm.rewieTab = true">Reviews</button>
+            <button class="btn" ng-class="{active: !vm.rewieTab}" type="button" name="button" ng-click="vm.rewieTab = false">Questions</button>
+            <div class="tabs-question" ng-show="!vm.rewieTab">
+              <form class="" action="index.html" method="post">
+                <input type="text" name="name" value="" placeholder="search a question, find an answer">
+                <button class="fa fa-search" type="button" name="button"></button>
+              </form>
+              <button class="btn" type="button" name="button" ng-click="vm.openQuestionModal({questions: vm.app.questions})">Ask a question</button>
+            </div>
+          </div>
+          <div class="col-md-12 testimonial-panel" ng-show="vm.rewieTab">
+            <div class="testimonial-average">
+              Average:
+              <i class="fa fa-star"></i>
+              <i class="fa fa-star"></i>
+              <i class="fa fa-star"></i>
+              <i class="fa fa-star"></i>
+              <i class="fa fa-star-o"></i>
+              (23 ratings)
+            </div>
+            <div class="testimonial-filter">
+              <select class="" name="">
+                <option>Filter by rating</option>
+              </select>
+              <button class="btn" type="button" name="button">Write a review</button>
+            </div>
+          </div>
+          <!-- Reviews -->
+          <div class="col-md-12" ng-repeat="(i, testimonial) in vm.app.testimonials | limitTo:2" ng-show="vm.isTestimonialShown(testimonial) && vm.rewieTab">
```

Comment:
Usually prefer to not add multiple conditionals in expressions. Maybe abstracting this logic to `vm.isReviewShown()` would be nice?